zetaclient/evm/evm_client.go

```diff
@@ -67,16 +71,15 @@ const (
 	TopicsDeposited    = 2 // [signature, asset]            https://github.com/zeta-chain/protocol-contracts/blob/d65814debf17648a6c67d757ba03646415842790/contracts/evm/ERC20Custody.sol#L42
 )
 
-// EVMChainClient represents the chain configuration for an EVM chain
+// ChainClient represents the chain configuration for an EVM chain
 // Filled with above constants depending on chain
-type EVMChainClient struct {
-	*ChainMetrics
+type ChainClient struct {
```

Comment:
same. Can we just keep `EVMChainClient` to differentiate from the interface name `ChainClient`?